packages/devtools-source-map/src/tests/source-map.js

```diff
@@ -0,0 +1,33 @@
+const fs = require("fs");
+const path = require("path");
+
+jest.mock("devtools-network-request");
+const { getOriginalURLs } = require("../source-map");
+
+function getMap(_path) {
+  const mapPath = path.join(__dirname, _path);
+  return fs.readFileSync(mapPath, "utf8");
+}
+
+describe("source maps", () => {
+  test("getOriginalURLs", async () => {
+    const source = {
```

Comment:
Hmm, that makes sense. 

I think for this test it's fine that we have a simple map file. For other tests it'd be nice to have webpack / browserify etc